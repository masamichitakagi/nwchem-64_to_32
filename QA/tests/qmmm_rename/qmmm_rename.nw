echo
memory total 800 Mb

start asa 

#prepare
#  system asa_ref
#  source asa.pdb
#  new_top new_seq 
#  new_rst
#  modify atom 2:_CB quantum
#  modify atom 2:2HB quantum
#  modify atom 2:3HB quantum
#  modify atom 2:_OG quantum
#  modify atom 2:_HG quantum
#  center
#  orient
#  solvate
#  fix segments beyond 0.5 2:_OG
#  update lists
#  ignore
#  write asa_ref.rst 
#  write asa_ref.pdb   # Write out PDB file to check structure
#end
#task prepare
#

#----------------------------------------
# this can be replaced by
# task shell "cp asa_ref.rst asa_qmmm.rst"
#-----------------------------------------
prepare
read asa_ref.rst
write asa_qmmm.rst
end
task prepare

md
  system asa_qmmm
  cells 2 2 2
end

basis "ao basis" 
  * library "6-31G*"
  C1 library "6-311++G**"
end

dft
 print low
 iterations 500
end

qmmm
rename name.dat
end

task qmmm dft energy

