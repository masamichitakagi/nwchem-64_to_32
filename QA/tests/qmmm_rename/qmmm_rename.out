refund: UserID = marat
refund: SLURM Job ID = 1033237
refund: Number of nodes          = 4
refund: Number of cores per node = 16
refund: Number of cores          = 64
refund: Amount of time requested = 30
refund: Directory = /home/marat/codes/nwchem-dev/QA/tests/qmmm_rename-save
 
Processor list
 
g[0-3]
 
 argument  1 = /home/marat/codes/nwchem-dev/QA/tests/qmmm_rename-save/qmmm_opt0.nw
 


============================== echo of input deck ==============================
echo
memory total 800 Mb

start asa

#prepare
#  system asa_ref
#  source asa.pdb
#  new_top new_seq
#  new_rst
#  modify atom 2:_CB quantum
#  modify atom 2:2HB quantum
#  modify atom 2:3HB quantum
#  modify atom 2:_OG quantum
#  modify atom 2:_HG quantum
#  center
#  orient
#  solvate
#  fix segments beyond 0.5 2:_OG
#  update lists
#  ignore
#  write asa_ref.rst
#  write asa_ref.pdb   # Write out PDB file to check structure
#end
#task prepare
#

#----------------------------------------
# this can be replaced by
# task shell "cp asa_ref.rst asa_qmmm.rst"
#-----------------------------------------
prepare
read asa_ref.rst
write asa_qmmm.rst
end
task prepare

md
  system asa_qmmm
  cells 2 2 2
end

basis "ao basis"
  * library "6-31G*"
  C1 library "6-311++G**"
end

dft
 print low
 iterations 500
end

qmmm
rename name.dat
end

task qmmm dft energy

================================================================================


                                         
                                         
 
 
              Northwest Computational Chemistry Package (NWChem) 6.3
              ------------------------------------------------------
 
 
                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352
 
                              Copyright (c) 1994-2013
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute
 
             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file
 
                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname        = g0
    program         = /scratch/nwchem
    date            = Thu Feb  6 15:11:07 2014

    compiled        = Thu_Feb_06_15:10:30_2014
    source          = /home/marat/codes/nwchem-dev
    nwchem branch   = Development
    nwchem revision = 25198
    ga revision     = 10467
    input           = /home/marat/codes/nwchem-dev/QA/tests/qmmm_rename-save/qmmm_opt0.nw
    prefix          = asa.
    data base       = ./asa.db
    status          = startup
    nproc           =       64
    time left       =     -1s



           Memory information
           ------------------

    heap     =   26214401 doubles =    200.0 Mbytes
    stack    =   26214401 doubles =    200.0 Mbytes
    global   =   52428800 doubles =    400.0 Mbytes (distinct from heap & stack)
    total    =  104857602 doubles =    800.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------
 
  0 permanent = .
  0 scratch   = .
 
 
 
 
                                NWChem Input Module
                                -------------------
 
 
                                  Prepare Module
                                  --------------

 Force field                           amber


 Directories used for fragment and segment files

                                       /home/marat/codes/nwchem/src/data/amber_s/
                                       /home/marat/codes/nwchem/src/data/amber_q/
                                       /home/marat/codes/nwchem/src/data/amber_x/
                                       /home/marat/codes/nwchem/src/data/amber_u/
                                       ./

 Parameter files used to resolve force field parameters

                                       /home/marat/codes/nwchem/src/data/amber_s/amber.par
                                       /home/marat/codes/nwchem/src/data/amber_q/amber.par
                                       /home/marat/codes/nwchem/src/data/amber_x/amber.par
                                       /home/marat/codes/nwchem/src/data/amber_u/amber.par
                                       ./amber.par
 Using existing topology               asa.top
 Topology                              asa.top

 Restart generation using command file directives

 Read restart                          asa_ref.rst


 Created restart                       asa_qmmm.rst


 Task  times  cpu:        0.0s     wall:        0.0s
 
 
                                NWChem Input Module
                                -------------------
 
 
  library name resolved from: environment
  library file name is: <
 /home/scicons/felicia/apps/nwchem/nwchem-6.3/src/basis/libraries/>
  


 Summary of "ao basis" -> "" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 C1                        6-311++G**               10       23   5s4p1d
 *                           6-31G*                   on all atoms 


 in qmmm_input:                     
 out of qmmm_input:                     
 ****************************************************
 *      NWChem QM/MM Interface Module               *
 *                                                  *
 *      Developed Marat Valiev 2004                 *
 ****************************************************
                    QM/MM Interface Parameters
                    --------------------------
 operation             task:operation        energy                                                        
 reference energy      qmmm:uqmatm               0.000000  <--- Warning zero value is not advisable !!!    
 bqzone radius         qmmm:bqzone               9.000000                                                  
 excluded bq's         qmmm:bq_exclude       none                                                          
 link atom type        qmmm:linkatm          hydrogens                                                     
 optimization method   qmmm:optimization     bfgs                                                          
 ecp on link atoms     qmmm:link_ecp         auto                                                          
 interface api         qmmm:interface        qm                                                            
 ------------------------------------------------------


 number of quantum atoms                     5
 Total number of Bq charges                   362
 Number of solute Bq charges                    29
 Number of solvent Bq charges                   333
 Total Bq charge:  -5.500000000000038E-002
                     Loading custom atom names from  name.dat
                     ----------------------------------------
         6
   
  C1            0.886894        1.685381       -0.084112        17
  H             1.896658        1.241348       -0.097263        18
  H             0.702993        2.180159       -1.045539        19
  OX            0.904933        2.735151        0.874459        20
  H             0.000000        2.992744        1.044729        21
  H_L           0.143504        0.878161        0.065495        15
 Total number of active Bq charges                     41
 frozen_density,esp F F
                     Loading custom atom names from  name.dat
                     ----------------------------------------
         6
   
  C1            0.886894        1.685381       -0.084112        17
  H             1.896658        1.241348       -0.097263        18
  H             0.702993        2.180159       -1.045539        19
  OX            0.904933        2.735151        0.874459        20
  H             0.000000        2.992744        1.044729        21
  H_L           0.143504        0.878161        0.065495        15
 
                                 NWChem DFT Module
                                 -----------------
 
 
                      Basis "ao basis" -> "ao basis" (cartesian)
                      -----
  C1 (Carbon)
  -----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  4.56324000E+03  0.001967
  1 S  6.82024000E+02  0.015231
  1 S  1.54973000E+02  0.076127
  1 S  4.44553000E+01  0.260801
  1 S  1.30290000E+01  0.616462
  1 S  1.82773000E+00  0.221006
 
  2 S  2.09642000E+01  0.114660
  2 S  4.80331000E+00  0.919999
  2 S  1.45933000E+00 -0.003031
 
  3 P  2.09642000E+01  0.040249
  3 P  4.80331000E+00  0.237594
  3 P  1.45933000E+00  0.815854
 
  4 S  4.83456000E-01  1.000000
 
  5 P  4.83456000E-01  1.000000
 
  6 S  1.45585000E-01  1.000000
 
  7 P  1.45585000E-01  1.000000
 
  8 S  4.38000000E-02  1.000000
 
  9 P  4.38000000E-02  1.000000
 
 10 D  6.26000000E-01  1.000000
 
 11 S  3.04752490E+03  0.001835
 11 S  4.57369510E+02  0.014037
 11 S  1.03948690E+02  0.068843
 11 S  2.92101550E+01  0.232184
 11 S  9.28666300E+00  0.467941
 11 S  3.16392700E+00  0.362312
 
 12 S  7.86827240E+00 -0.119332
 12 S  1.88128850E+00 -0.160854
 12 S  5.44249300E-01  1.143456
 
 13 P  7.86827240E+00  0.068999
 13 P  1.88128850E+00  0.316424
 13 P  5.44249300E-01  0.744308
 
 14 S  1.68714400E-01  1.000000
 
 15 P  1.68714400E-01  1.000000
 
 16 D  8.00000000E-01  1.000000
 
  H (Hydrogen)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  1.87311370E+01  0.033495
  1 S  2.82539370E+00  0.234727
  1 S  6.40121700E-01  0.813757
 
  2 S  1.61277800E-01  1.000000
 
  OX (Oxygen)
  -----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  5.48467170E+03  0.001831
  1 S  8.25234950E+02  0.013950
  1 S  1.88046960E+02  0.068445
  1 S  5.29645000E+01  0.232714
  1 S  1.68975700E+01  0.470193
  1 S  5.79963530E+00  0.358521
 
  2 S  1.55396160E+01 -0.110778
  2 S  3.59993360E+00 -0.148026
  2 S  1.01376180E+00  1.130767
 
  3 P  1.55396160E+01  0.070874
  3 P  3.59993360E+00  0.339753
  3 P  1.01376180E+00  0.727159
 
  4 S  2.70005800E-01  1.000000
 
  5 P  2.70005800E-01  1.000000
 
  6 D  8.00000000E-01  1.000000
 
  H_L (Hydrogen)
  --------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  1.87311370E+01  0.033495
  1 S  2.82539370E+00  0.234727
  1 S  6.40121700E-01  0.813757
 
  2 S  1.61277800E-01  1.000000
 


 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 C1                   modified:6-311++G**           16       38   8s6p2d
 H                           6-31G*                  2        2   2s
 OX                          6-31G*                  6       15   3s2p1d
 H_L                         6-31G*                  2        2   2s


 
  Caching 1-el integrals 
 Nuclear repulsion energy =   40.2316466749587     
 Bq nuclear interaction energy =  0.167173616859156     
 
 WARNING : Found     1 linear dependencies 
 S eigenvalue threshold:     1.00000E-05
 Smallest S eigenvalue :     5.82525E-06
 Largest  S eigenvalue :     5.82525E-06
 
 Nuclear repulsion energy =   40.2316466749587     
 Bq nuclear interaction energy =  0.167173616859156     
 Nuclear repulsion energy =   40.2316466749587     
 Bq nuclear interaction energy =  0.167173616859156     
   Time after variat. SCF:      0.8
   Time prior to 1st pass:      0.8
 Nuclear repulsion energy =   40.2316466749587     
 Bq nuclear interaction energy =  0.167173616859156     
 Nuclear repulsion energy =   40.2316466749587     
 Bq nuclear interaction energy =  0.167173616859156     
 Nuclear repulsion energy =   40.2316466749587     
 Bq nuclear interaction energy =  0.167173616859156     
 Nuclear repulsion energy =   40.2316466749587     
 Bq nuclear interaction energy =  0.167173616859156     
 Nuclear repulsion energy =   40.2316466749587     
 Bq nuclear interaction energy =  0.167173616859156     
 Nuclear repulsion energy =   40.2316466749587     
 Bq nuclear interaction energy =  0.167173616859156     
 Nuclear repulsion energy =   40.2316466749587     
 Bq nuclear interaction energy =  0.167173616859156     
 Nuclear repulsion energy =   40.2316466749587     
 Bq nuclear interaction energy =  0.167173616859156     
 Nuclear repulsion energy =   40.2316466749587     
 Bq nuclear interaction energy =  0.167173616859156     
 Nuclear repulsion energy =   40.2316466749587     
 Bq nuclear interaction energy =  0.167173616859156     
 Nuclear repulsion energy =   40.2316466749587     
 Bq nuclear interaction energy =  0.167173616859156     
 Nuclear repulsion energy =   40.2316466749587     
 Bq nuclear interaction energy =  0.167173616859156     
 Nuclear repulsion energy =   40.2316466749587     
 Bq nuclear interaction energy =  0.167173616859156     
 Nuclear repulsion energy =   40.2316466749587     
 Bq nuclear interaction energy =  0.167173616859156     
  Resetting Diis
 Nuclear repulsion energy =   40.2316466749587     
 Bq nuclear interaction energy =  0.167173616859156     
 Nuclear repulsion energy =   40.2316466749587     
 Bq nuclear interaction energy =  0.167173616859156     
 Nuclear repulsion energy =   40.2316466749587     
 Bq nuclear interaction energy =  0.167173616859156     
 Nuclear repulsion energy =   40.2316466749587     
 Bq nuclear interaction energy =  0.167173616859156     
 Nuclear repulsion energy =   40.2316466749587     
 Bq nuclear interaction energy =  0.167173616859156     


         Total DFT energy =     -114.795251273854
      One electron energy =     -236.916526521550
           Coulomb energy =       96.229051732272
    Exchange-Corr. energy =      -14.506596776394
 Nuclear repulsion energy =       40.398820291818

 Numeric. integr. density =       18.000000044681

     Total iterative time =      0.8s


 Nuclear repulsion energy =   40.2316466749587     
 Bq nuclear interaction energy =  0.167173616859156     
 Bq nuclear interaction energy =  0.167173616859154     
------------------------------------------------------------------------
 QM/MM Energy
------------------------------------------------------------------------
 quantum energy                      -114.795251274 (-.301395E+06  kjoule/mol)
 quantum energy adjusted             -114.795251274 (-.301395E+06  kjoule/mol)
 quantum energy internal             -114.783648488 (-.301364E+06  kjoule/mol)
 Bq-nuclear energy                      0.167173617 (0.438914E+03  kjoule/mol)
 Bq-electron energy                    -0.178776403 (-.469377E+03  kjoule/mol)
 classical energy                       0.735231966 (0.193035E+04  kjoule/mol)
 total qmmm energy                   -114.060019308 (-.299465E+06  kjoule/mol)
------------------------------------------------------------------------
 

 Task  times  cpu:        1.2s     wall:       13.5s
 
 
                                NWChem Input Module
                                -------------------
 
 
 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:  297      297     2716     1474      490        0        0       44     
number of processes/call 3.66e+00 1.76e+00 7.56e+00 0.00e+00 0.00e+00
bytes total:             1.37e+07 3.57e+06 5.10e+06 0.00e+00 0.00e+00 3.52e+02
bytes remote:            8.10e+06 1.08e+06 3.54e+06 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 466768 bytes
 
MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        59	       131
	current total bytes		         0	         0
	maximum total bytes		 210956936	 393755376
	maximum total K-bytes		    210957	    393756
	maximum total M-bytes		       211	       394
 
 
                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:
 
                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018
 
                              AUTHORS & CONTRIBUTORS
                              ----------------------
          E. Apra, E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski,
       T. P. Straatsma, M. Valiev, H. J. J. van Dam, D. Wang, T. L. Windus,
        J. Hammond, J. Autschbach, K. Bhaskaran-Nair, J. Brabec, K. Lopata,
     F. Aquino, S. Hirata, M. T. Hackler, T. Risthaus, M. Malagoli,J. Mullin,
        P. Nichols, R. Peverati, J. Pittner, Y. Zhao, P.-D. Fan, A. Fonari,
     R. J. Harrison, M. Dupuis, D. Silverstein, D. M. A. Smith, J. Nieplocha,
        V. Tipparaju, M. Krishnan, B. E. Van Kuiken, A. Vazquez-Mayagoitia,
        L. Jensen, M. Swart, Q. Wu, T. Van Voorhis, A. A. Auer, M. Nooijen,
      L. D. Crosby, E. Brown, G. Cisneros, G. I. Fann, H. Fruchtl, J. Garza,
   K. Hirao, R. Kendall, J. A. Nichols, K. Tsemekhman, K. Wolinski, J. Anchell,
       D. Bernholdt, P. Borowski, T. Clark, D. Clerc, H. Dachsel, M. Deegan,
        K. Dyall, D. Elwood, E. Glendening, M. Gutowski, A. Hess, J. Jaffe,
   B. Johnson, J. Ju, R. Kobayashi, R. Kutteh, Z. Lin, R. Littlefield, X. Long,
        B. Meng, T. Nakajima, S. Niu, L. Pollack, M. Rosing, K. Glaesemann,
      G. Sandrone, M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe, A. Wong,
                                     Z. Zhang.

 Total times  cpu:        1.7s     wall:       21.4s
